File: tokilearning/Latihan/case.cpp
#include <stdio.h>
#include <stdlib.h>

using namespace std;

int n;

int main(){
    scanf("%d",&n);
    if (n<10) printf("satuan\n");
      else if (n<100) printf("puluhan\n");
        else if (n<1000) printf("ratusan\n");
          else if (n<10000) printf("ribuan\n");
            else printf("puluhribuan\n");
    //system("pause");
    return 0;
}

File: tokilearning/Latihan/case.pas
var
  n:longint;

begin
  readln(n);
  case n of
    1..9: writeln('satuan');
    10..99: writeln('puluhan');
    100..999: writeln('ratusan');
    1000..9999: writeln('ribuan');
    10000..30000: writeln('puluhribuan');
  end;
end.
